problems/sortedClassesBySize.js

```diff
@@ -12,7 +12,9 @@
 */
 
 function sortClassesBySize() {
-
+    let arr = [];
+    for (let i = 0; i <= arr.length; i++)
+    return arr[i].sort("");
```

Comment:
Dont forget the `{}` for the for-loops also remember the behavior of return and what happens if we write it inside of a for-loop. For this question you have the right idea to use sort but it should not be used within a for-loop. After sort then you want to access the objects keys for the subject by iterating through the array of object and return an array of strings, a good method for this is the array map method